src/modules/ROOT/pages/changelog/202303/07.json

```diff
@@ -0,0 +1,122 @@
+{
+  "meta": {
+    "releasedOn": "2023.09.28"
+  },
+  "added": [
+    {
+      "what": [
+        "the option to delete a preset filter from filter preset dropdown in a record list."
+      ],
+      "why": [
+        "to provide the user with the option to delete a preset filter."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1449/commits/e7c9c857dc9c848edc75ddb5475745f4cef02e14"
+      ]
+    },
+    {
+      "what": [
+        "the option to sort the system fields `ownedBy`, `createdBy`, `updatedBy`, `deletedBy` and the fields user and record in a record list."
+      ],
+      "why": [
+        "to make sorting/viewing data easier for the user."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1449/commits/e73119a888474ef397f837cea854d29b15cf2035"
+      ]
+    }
+  ],
+  
+  "changed": [
+    {
+      "what": [
+        "option to clone a page layout by adding two sub options - for cloning a page with page block reference and for cloning a page without page block reference.",
+        "In the first case, configuration changes that are made to the original page's blocks are also reflected to the copied page's page blocks.",
+        "As for the other option, the changes made to the original page's blocks are not reflected to the copied page's page blocks."
```

Comment:
Prefer to refer to these options by _name_ rather by sequence number.
For example; `When cloning by reference, configuration changes made to the original ...`

This might be a personal preference but I do think it reads a bit better.